07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md

```diff
@@ -29,7 +29,8 @@ POST https://<WebAPI-URL>/adhocscan/subscription/{subscriptionId}/ControlScanRes
 ```
 > _Note:_<br/>
 > _1. 'WebAPI-URL' varies per AzTS setup. You need to reach out to AzTS admin to get value for `<WebAPI-URL`> as mentioned [here](../README.md#setup-for-azts-admin-only)._<br/>
-> _2. 'api-version' is a required parameter. Current supported version: **1.0**_
+> _2. API will return 100 records of control scan result at a time. In order to get next set of records, you have to pass skipToken (available in API response) as part of next API request._<br/>
```

Comment:
As we have added details about require headers, request body can we details about the response bcz now its a complex object and not direct data.

Reply:
Added API response with details in proper tabular format.